docs/examples.rst

```diff
@@ -1,4 +1,10 @@
 Examples
 ========
 
-TODO: document here
+You shall take a look at folium's example gallery_.
+
+.. _gallery: http://nbviewer.jupyter.org/github/python-visualization/folium/tree/master/examples/
+
+If this is not enough, you may find fancier examples here_.
+
+.. _here: http://nbviewer.jupyter.org/github/ocefpaf/folium_notebooks/tree/master/
```

Comment:
I must merge these two into one but I am reluctant to inflate folium's repository with many notebooks.  Maybe a new rep, under `python-visualization` org is in order.